docs/docs/genai/eval-monitor/scorers/third-party/ragas.mdx

```diff
@@ -0,0 +1,151 @@
+
+### General Purpose
+
+Flexible evaluation metrics for various use cases:
+
+| Scorer                                                                             | What does it evaluate?                             | RAGAS Docs                                                                                                                             |
+| ---------------------------------------------------------------------------------- | -------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------- |
+| <APILink fn="mlflow.genai.scorers.ragas.AspectCritic">AspectCritic</APILink>       | Evaluates specific aspects of the output using LLM | [Link](https://docs.ragas.io/en/stable/concepts/metrics/available_metrics/aspect_critic/)                                              |
+| <APILink fn="mlflow.genai.scorers.ragas.RubricsScore">RubricsScore</APILink>       | Scores output based on predefined rubrics          | [Link](https://docs.ragas.io/en/stable/concepts/metrics/available_metrics/general_purpose/#rubrics-based-criteria-scoring)             |
+| <APILink fn="mlflow.genai.scorers.ragas.InstanceRubrics">InstanceRubrics</APILink> | Scores output based on instance-specific rubrics   | [Link](https://docs.ragas.io/en/stable/concepts/metrics/available_metrics/general_purpose/#instance-specific-rubrics-criteria-scoring) |
+
+### Other Tasks
+
+Specialized metrics for specific tasks:
+
+| Scorer                                                                                   | What does it evaluate?        | RAGAS Docs                                                                                      |
+| ---------------------------------------------------------------------------------------- | ----------------------------- | ----------------------------------------------------------------------------------------------- |
+| <APILink fn="mlflow.genai.scorers.ragas.SummarizationScore">SummarizationScore</APILink> | Quality of text summarization | [Link](https://docs.ragas.io/en/stable/concepts/metrics/available_metrics/summarization_score/) |
+
+## Creating Scorers by Name
+
+If a particular RAGAS metric is not listed above, you can create it dynamically using <APILink fn="mlflow.genai.scorers.ragas.get_scorer">get_scorer</APILink>:
+
+```python
+from mlflow.genai.scorers.ragas import get_scorer
+
+# Create scorer by name
+scorer = get_scorer(
+    metric_name="Faithfulness",
+    model="openai:/gpt-4",
+)
+
+feedback = scorer(trace=trace)
+```
+
+## Configuration
+
+RAGAS scorers accept metric-specific parameters. Any additional keyword arguments are passed directly to the RAGAS metric constructor:
```

Comment:
Could we add an example of instantiating a RAGAS scorer that requires additional kwargs?

Reply:
I actually tried finding a metric for this and didn't see any - did you have one in mind?